find.go

```diff
@@ -233,7 +233,13 @@ func (s *server) doFind(ctx context.Context, method, source string, req *url.URL
 			stats.WithMeasurements(metrics.FindLoad.M(1)))
 	}()
 
-	sg := &scatterGather[Backend, *model.FindResponse]{
+	// sgResponse is a struct that exists to capture the backend that the response has been received from
+	type sgResponse struct {
+		r *model.FindResponse
+		b Backend
```

Comment:
Since these variables are not used immediately nearby, more than a single-letter name is preferred. Suggest:
Maybe `rsp` and `bknd`?

Reply:
Makes sense - changed